src/physics/cam/cam_diagnostics.F90

```diff
@@ -683,6 +691,14 @@ subroutine diag_init_moist(pbuf2d)
       call add_default ('CUSH&IC    ',0, 'I')
       call add_default ('KVH&IC     ',0, 'I')
       call add_default ('KVM&IC     ',0, 'I')
+
+      !GC add these variables to the i.c. for spreads. DA radiances, RTTOV needs
+      !these.
+      call add_default ('TS      ', 0, 'I')
+      call add_default ('TREFHT  ', 0, 'I')
+      call add_default ('LANDFRAC', 0, 'I')
+      call add_default ('OCNFRAC ', 0, 'I')
```

Comment:
Why aren't these `IC` variables?
```suggestion
      !GC add these variables to the i.c. for spreads. DA radiances, RTTOV needs
      !these.
      call add_default ('TS&IC   ', 0, 'I')
      call add_default ('TREFHT&IC', 0, 'I')
      call add_default ('LANDFRAC&IC', 0, 'I')
      call add_default ('OCNFRAC&IC', 0, 'I')
```

Reply:
This is expected, you do not have an `addfld` call for these fields (see my comment below).